tools/two-machine-perf.ps1

```diff
@@ -44,8 +48,19 @@ Write-Output "Local address: $LocalAddress"
 
 $LocalInterface = (Get-NetIPAddress -IPAddress $LocalAddress -ErrorAction SilentlyContinue).InterfaceIndex
 $LocalMacAddress = (Get-NetAdapter -InterfaceIndex $LocalInterface).MacAddress
+$LocalVfAdapter = Get-NetAdapter | Where-Object {$_.MacAddress -eq $LocalMacAddress -and $_.ifIndex -ne $LocalInterface}
 Write-Output "Local interface: $LocalInterface, $LocalMacAddress"
 
+$LowestInterface = $LocalInterface
+if ($LocalVfAdapter) {
+    $script:LowestInterface = $LocalVfAdapter.ifIndex
+}
+
+$ChunkSize = 2048
```

Comment:
Some comments would be helpful here. What is a "chunk" and "batch"?

Reply:
Yeah, this is AF_XDP jargon (identical to terms used in Linux) so added a brief description.